src/main/java/duke/action/MyList.java

```diff
@@ -0,0 +1,43 @@
+package duke.action;
+
+import duke.task.Task;
+
+
+/**
+ * Represents an action to display the list of tasks.
+ */
+public class MyList implements Action {
+
+    /**
+     * The task list to be displayed.
+     */
+    private final TaskList mylist;
```

Comment:
Wouldn't something like tasks be more appropriate  according to the coding standard?